app/editor/src/store/hooks/storage/useStorage.ts

```diff
@@ -14,7 +14,7 @@ interface IStorageController {
     onUploadProgress?: (progressEvent: any) => void,
   ) => Promise<IItemModel>;
   download: (locationId: number, path: string) => Promise<unknown>;
-  stream: (locationId: number, path: string) => Promise<unknown>;
+  stream: (locationId: number, path: string) => Promise<string>;
```

Comment:
Why can't we update the type to `byte[]`

Reply:
In this case we will be retrieving an object URL for the media instead of getting its content directly.

Reply:
We won't be retrieving a URL from the stream endpoint.  We retrieve a stream of data.

Reply:
Yeah, strictly speaking we retrieve a stream of data based on the endpoint, and we will be using its object URL from the result for the case here as well. Thanks